### PR TITLE
Add deposit to kamino-sdk

### PR DESCRIPTION
**Pull request by peroxy**

### CODE REVIEW

File: packages/kamino-sdk/src/Kamino.ts [hunk 1]
```diff
@@ -495,6 +504,69 @@ export class Kamino {
...
   }
+
+  /**
+   * Get transaction instruction to deposit token A and B into a strategy.
+   * @param strategy Kamino strategy public key or on-chain object
+   * @param amountA Amount of token A to deposit into strategy
+   * @param amountB Amount of token B to deposit into strategy
+   * @param owner Owner (wallet, shareholder) public key
+   */
+  async deposit(strategy: PublicKey | StrategyWithAddress, amountA: Decimal, amountB: Decimal, owner: PublicKey) {
```
Comment (y2kappa) [inline packages/kamino-sdk/src/Kamino.ts:515]: annotate return type

File: packages/kamino-sdk/README.md [hunk 2]
```diff
@@ -169,4 +169,69 @@ tx = await assignBlockInfoToTransaction(connection, tx, signer.publicKey);
...
+  getAssociatedTokenAddressAndData,
+  createTransactionWithExtraBudget,
+  getCreateAssociatedTokenAccountInstructionsIfNotExist,
+  assignBlockInfoToTransaction
+} from '@hubbleprotocol/kamino-sdk';
+import Decimal from 'decimal.js';
+
+// setup Kamino SDK
+const strategyPubkey = new PublicKey('2H4xebnp2M9JYgPPfUw58uUQahWF8f1YTNxwwtmdqVYV'); // you may also fetch strategies from hubble config
+const owner = new PublicKey('HrwbdQYwSnAyVpVHuGQ661HiNbWmGjDp5DdDR9YMw7Bu'); // wallet with shares
```
Comment (y2kappa) [inline packages/kamino-sdk/README.md:191]: why does the wallet need to have shares already?

File: packages/kamino-sdk/README.md [hunk 3]
```diff
@@ -169,4 +169,69 @@ tx = await assignBlockInfoToTransaction(connection, tx, signer.publicKey);
...
+const connection = new Connection(clusterApiUrl('mainnet-beta'));
+const kamino = new Kamino('mainnet-beta', connection);
+// setup fee payer (wallet) that will sign the transaction 
+const signer = Keypair.generate();
```
Comment (y2kappa) [inline packages/kamino-sdk/README.md:195]:
```ts
let owner = new Uint8Array([
      206, 154, 73, 87, 65, 199, 92, 146, 120, 77, 160, 139, 166, 146, 124, 248,
      206, 138, 16, 20, 136, 213, 174, 49, 138, 87, 232, 216, 236, 124, 192,
      171, 65, 163, 50, 79, 255, 117, 9, 198, 50, 104, 226, 130, 215, 212, 111,
      16, 6, 96, 40, 144, 147, 189, 21, 21, 174, 16, 28, 82, 201, 39, 121, 16,
    ]);
    
 ```

File: packages/kamino-sdk/README.md (hunk 2 above)
Comment (peroxy) [reply packages/kamino-sdk/README.md:191]: Ah, I copied the sample from withdraw function, it doesnt need existing shares

**Pull request merged**